Microsoft.Toolkit.Uwp.UI.Controls/DropShadowPanel/DropShadowPanel.xaml

```diff
@@ -1,11 +1,20 @@
-﻿<UserControl x:Class="Microsoft.Toolkit.Uwp.UI.Controls.DropShadowPanel"
-             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-             mc:Ignorable="d">
-    <Grid>
-        <Border x:Name="ShadowElement" />
-        <ContentPresenter Content="{x:Bind CastingElement}" />
-    </Grid>
-</UserControl>
+<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls">
+
+    <Style TargetType="controls:DropShadowPanel">
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="controls:DropShadowPanel">
+                    <Grid Background="{TemplateBinding Background}"
+                          BorderBrush="{TemplateBinding BorderBrush}"
+                          BorderThickness="{TemplateBinding BorderThickness}">
+                        <Border x:Name="ShadowElement" />
+                        <ContentPresenter x:Name="CastingElement"
+                                          Content="{TemplateBinding Content}" />
```

Comment:
No need to bind content. Happens automatically for ContentPresenters within a ControlControl